simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/StationComponent.java

```diff
@@ -94,4 +96,28 @@ public SetVariableValidationResult validate(SetVariableDatum setVariableDatum) {
         return new SetVariableValidationResult(setVariableDatum, validationResult);
     }
 
+    /**
+     * Generates report data for all variables in the component
+     *
+     * @param onlyMutableVariables if true, returns only those variables that can be set by the operator
+     * @return list of {@link ReportDatum}
+     */
+    public List<ReportDatum> generateReportData(boolean onlyMutableVariables) {
```

Comment:
Its there a unit test that covers this method?

Reply:
there is now